src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -134,6 +134,12 @@ The SDK sets the Span operation to `app.start.cold` for *Cold start_* and `app.s
 
 The SDK uses the `SentryPerformanceProvider` (ContentProvider) creation time as the beginning of the app start and the first `Activity#onResume` call as the end.
 
+<Note>
+
+The App Start Instrumentation is only measured if the Process is of the importance [RunningAppProcessInfo.IMPORTANCE_FOREGROUND](https://developer.android.com/reference/android/app/ActivityManager.RunningAppProcessInfo#IMPORTANCE_FOREGROUND), that means the process is running the foreground UI.
```

Comment:
maybe worth mentioning that when the Process is opened via background work, it's not measured, since it's not running the foreground UI? or is it obvious?

Reply:
Yeah, good idea. Maybe also give an example of the notification case?